content/docs/getting-started.md

```diff
@@ -0,0 +1,101 @@
+---
+id: getting-started
+title: Getting Started
+permalink: docs/getting-started.html
+next: add-react-to-a-website.html
+redirect_from:
+  - "docs/"
+  - "docs/index.html"
+  - "docs/getting-started-ko-KR.html"
+  - "docs/getting-started-zh-CN.html"
+  - "docs/installation.html"
+  - "download.html"
+  - "downloads.html"
+  - "docs/try-react.html"
+  - "docs/tooling-integration.html"
+  - "docs/package-management.html"
+  - "docs/language-tooling.html"
+  - "docs/environments.html"
+---
+
+This page is an overview of the React documentation and related resources.
```

Comment:
maybe a note about what React is, linking back to the homepage? in case someone misses it